components/src/Guidelines/index.tsx

```diff
@@ -0,0 +1,47 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui'
+import React, { ReactNode } from 'react'
+import { Flex, Divider, Box, Text as ThemeUIText } from 'theme-ui'
+
+export const Guidelines = ({ children }: Prop) => <Box>{children}</Box>
+
+const Dos = ({ children, icon = '✅', title = 'Dos' }: Props) => (
+  <GuidelinesList icon={icon} title={title} children={children} />
+)
+
+const Donts = ({ children, icon = '❌', title = 'Donts' }: Props) => (
+  <GuidelinesList icon={icon} title={title} children={children} />
+)
+
+
+const GuidelinesList = ({ children, icon, title }: Props) => {
+
+    return (
+  <Box variant = 'guidelines.list' >
+    <Flex>
+      {icon}
+      <ThemeUIText
+        as="h3"
+        variant = 'guidelines.titles'
+      >
+        {title}
+      </ThemeUIText>
+    </Flex>
+    <ul sx= {{listStyleType: "none", paddingX: '30px' }}>{children.map(child => <React.Fragment>{child}<Divider variant ='guidelines.divider'/></React.Fragment> )}</ul>
```

Comment:
You can change this `paddingX` prop for `paddingLeft`, because there's no padding on the right side. And I think 28px is closer to the value used :sparkles:

Reply:
Right, It's corrected! :rocket: